File: Python/examples/prostate/analysis.py
import matplotlib as mpl
import pandas as pd

# Set matplotlib settings
mpl.get_backend()
mpl.use('TkAgg')
import matplotlib.pyplot as plt
plt.style.use('ggplot')

from easyml.factory import easy_glmnet


if __name__ == "__main__":
    # Load data
    prostate = pd.read_table('./prostate.txt')

    # Analyze data
    easy_glmnet(prostate, 'lpsa',
                alpha=1, n_lambda=200, standardize=False, cut_point=0, max_iter=1e6)

File: R/inst/examples/prostate/analysis.R
library(easyml) # https://github.com/CCS-Lab/easyml

# Load data
data("prostate", package = "easyml")

# Analyze data
easy_glmnet(prostate, "lpsa", 
            n_samples = 100, n_divisions = 50, n_iterations = 5, 
            random_state = 1, n_core = 8)

# Analyze data
system.time(easy_glmnet(prostate, "lpsa", 
                        n_samples = 500, n_divisions = 500, n_iterations = 1, 
                        random_state = 1, n_core = 1)
)

# Analyze data
system.time(easy_glmnet(prostate, "lpsa", 
                        n_samples = 500, n_divisions = 500, n_iterations = 1, 
                        random_state = 1, n_core = 8)
)
